Framework/DataHandling/src/LoadInstrument.cpp

```diff
@@ -99,6 +102,32 @@ void LoadInstrument::exec() {
   m_filename = getPropertyValue("Filename");
```

Comment:
Minor point (I know you didn't add this), but I dislike when algorithms set member variables when they don't need to. At the point that `execute` is called we are in a functional process. i.e. there is no need for encapsulated state. Would be nicer to avoid these and have these as local variables.